src/httpurr/client/node.cljs

```diff
@@ -21,13 +22,13 @@
      (when qs {:query qs})
      (when qp {:query (.stringify querystring (clj->js qp))}))))
 
-(deftype HttpResponse [msg]
+(deftype HttpResponse [msg body]
```

Comment:
I don't think this is a breaking change. I couldn't find a better to get the body here.